csharp/src/Vista.SDK/LocalId.cs

```diff
@@ -70,5 +72,29 @@ public bool Equals(LocalId? other)
 
     public override string ToString() => _builder.ToString();
 
+    public static LocalId Parse(string localIdStr, out LocalIdErrorBuilder errorBuilder) =>
+        LocalIdBuilder.Parse(localIdStr, out errorBuilder).Build();
+
     public static LocalId Parse(string localIdStr) => LocalIdBuilder.Parse(localIdStr).Build();
 }
+
+public enum ParsingState
```

Comment:
Should this be internal?

Reply:
I don't think we should separate since we want to check which state the error happened.